.changeset/slick-badgers-go.md

```diff
@@ -0,0 +1,12 @@
+---
+"@fluidframework/presence": minor
+---
+---
+"section": feature
+---
+
+`off` event deregistration pattern now supported
+
+Event subscriptions  within `@fluidframework/presence` may now use `off` to deregister event listeners, including initial listeners provided to `Notifications`.
+
+Some type names have shifted within the API though no direct outside of package is expected. The most visible rename is `NotificationSubscribable` to `NotificationListenable`. Other shifts are to use types now exported thru `@fluidframework/core-interfaces` where the most notable is `ISubscribable` is now `Listenable`.
```

Comment:
I _think_ the first sentence is trying to say "no direct action outside of the package is expected" but not sure if the type name changes could affect consumers or if they shouldn't be referencing the types directly anyway.

If my thinking is correct, I'd suggest "no impact is expected to consumers of this package except updating type imports in case the types were being referenced directly" or something along those lines.